backend/app/repositories/BaseReactiveMongoRepository.scala

```diff
@@ -55,9 +55,8 @@ trait BaseRepository[T <: BaseEntity[ID], ID <: BaseId[_]]
   def failoverStrategy: FailoverStrategy =
     FailoverStrategy(initialDelay = 100 milliseconds, retries = 16)
 
-  def upsert(t: T)(implicit
-      writer: Writes[ID],
-      dbSession: DBSession): Future[BSONObjectID]
+  def upsert(
```

Comment:
hint: formatting looks a bit odd (same below)

Reply:
I did run the formatter...without a change